AttoBot/lambda_function.py

```diff
@@ -25,6 +25,12 @@
 
 SECRET = os.environ["SECRET"]
 
+TAG_REQ = "\n".join((
+    "Please make sure that:",
+    "- [ ] CI passes for supported Julia versions (if applicable).",
```

Comment:
It's getting there pretty quickly. We should ask @nkottary to add a pending status indicator when the job gets triggered so we don't overlook it.

Both are useful to look at. JuliaCIBot isn't going to do well with packages that need custom setup, depend on non-default programs or libraries, etc. Those are usually easier to set up by a package author in their own .travis.yml and are likely non-trivial to replicate completely generally.